catalog.html

```diff
@@ -1,264 +1,293 @@
-                    <img class="product-card-image" src="#" alt="">
-                    <h3 class="product-card-title">GreenTree Inn</h3>
-                  </a>
-               <span class="inner-header-title">Гостиница></span>
-                  <b class="product-card-price">От 1500 ₽</b>
-                  <a class="product-card-button button" href="#">Подробнее</a>
-                  <a class="product-card-button button" href="#">В Избранное</a>
-                </li>
-              </ul>
-
-        <ul class="pagination">
-        <li class="pagination-item">
-          <a class="pagination-link pagination-current">1</a>
-        </li>
-
-        <li class="pagination-item">
-          <a class="pagination-link" href="#">2</a>
-        </li>
-
-        <li class="pagination-item">
-          <a class="pagination-link" href="#">3</a>
-        </li>
-
-        <li class="pagination-item">
-          <a class="pagination-link" href="#">4</a>
-        </li>
-
-        <li class="pagination-item">
-          <a class="pagination-link" href="#">10</a>
-        </li>
-       </ul>
-
-       <section>
-        <h2 class="page-footer-title">Подпишитесь на рассылку</h2>
-        <p class="advantages-description">Только полезная информация и никакого спама,честное бойскаутское!</p>
-        <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
-          <label for="newsletter-email">Email</label>
-          <input class="field" placeholder="Ваш e-mail" type="email" name="newsletter-email" id="newsletter-email"
-            required>
-
-      <button class="button newsletter-button" type="submit">Подписаться</button>
-        </form>
-       </section>
-
-      </main>
-
-       <footer>
-      <section class="footer social">
-        <h2 class="page-footer-title">Контакты</h2>
-        <ul class="footer-social-list">
-          <li class="footer-social-item">
-            <a class="button" href="#">
-              <span class="visually-hidden">ВКонтакте</span>
-            </a>
-          </li>
-          <li class="footer-social-item">
-            <a class="button" href="#">
-              <span class="visually-hidden">Твиттер</span>
-            </a>
-          </li>
-
-
-          <li class="footer-social-item">
-            <a class="button" href="#"></a>
-            <span class="visually-hidden">Youtube</span>
-            </li>
-          </ul>
-        </section>
-
-      <a class="footer-contacts-address-phone" href="tel:78128121212">+7 (812) 812-12-12</a>
-      <p class="about-text"><a class="htmlacademy-link" href="https://htmlacademy.ru/"></a>.html academy</p>
-    </footer>
-
-     </body>
-     </html>
+<!DOCTYPE html>
+<html lang="ru">
+
+  <head>
+    <meta charset="utf-8">
+    <title>Каталог</title>
+
+    <body>
+  <header class="page-header">
+    <a class="navigation-logo" href="#">
+      <img class="page-header-logo" src="#" width="140" height="70" alt="Логотип города Седона.">
```

Comment:
Нигде не подключены изображения, в src стоит `#` вместо пути к файлу изображения. Должно быть так:
```suggestion
      <img class="page-header-logo" src="images/logo.svg" width="140" height="70" alt="Логотип города Седона.">
```

Замечание пишу только здесь, исправить нужно везде.

Чтобы понять, что не так, стоит посмотреть опубликованную версию, ссылку на которую можно узнать на странице задания:
![image](https://github.com/htmlacademy-htmlcss/2448503-sedona-1/assets/17790771/12b91a00-7e7d-4f68-a310-377adae9bfaa)